internal/view/assets/less/common.less

```
@@ -11,17 +11,16 @@
 	--main: #f44336;
 	--errorColor: #f44336;
 	--selectedBg: #ffe7e5;
+}
 
-	@media (prefers-color-scheme: dark) {
-		&:root {
-			--bg: #1f1f1f;
-			--headerBg: #292929;
-			--contentBg: #292929;
-			--border: #191919;
-			--color: #fff;
-			--selectedBg: #261918;
-		}
-	}
+select {
+	background: var(--contentBg);
+	border: 1px solid var(--border);
+	border-radius: 4px;
+	color: var(--color);
+}
+#version {
+	color: var(--color);
```

Matching my suggestion below.

```suggestion
}

footer {
	color: var(--color);
```